minimum-constraints.txt

```diff
@@ -44,7 +44,7 @@ wheel===0.29.0
 
 # Direct dependencies for runtime (must be consistent with requirements.txt)
 
-pywbem===0.12.0
+pywbem===0.12.5
```

Comment:
In addition to increasing the minimum version in this file, it should also be increased in requirements.txt: `pywbem>=0.12.5`.

Reply:
I caught this one after I submitted the original pr. Fixed.